app/forms/support_interface/application_forms/course_search_form.rb

```diff
@@ -0,0 +1,12 @@
+module SupportInterface
+  module ApplicationForms
+    class CourseSearchForm
+      include ActiveModel::Model
+      include ValidationUtils
+
+      attr_accessor :course_code
+
+      validates :course_code, presence: true
```

Comment:
Could we add an error message for this of "Enter a course code" rather than the default "can't be blank"?